doc/video/install.rst

```diff
@@ -0,0 +1,284 @@
+Video script: Installation
+**************************
+
+Introduction
+============
+
+.. note::
+
+   Use a totally fresh install of Anaconda.
+   This includes removing conda-forge from the channels list, so there is no warning message in Step 6.
+
+   Open the install documentation at https://docs.messageix.org.
+   Open https://github.com/iiasa/message_ix in a separate tab, for later.
+
+Hello everybody and welcome to this video tutorial on the installation of MESSAGEix through Anaconda on Windows.
+
+Here, we are looking at the Installation page of the MESSAGEix documentation at docs dot messageix dot org.
```

Comment:
```suggestion
You can also find on the Installation page of the MESSAGEix documentation at docs dot messageix dot org, shown here. It is also linked in the description below. While you are there, make sure to check the prerequisite knowledge and skills to use MESSAGEix, as this will not be covered in our tutorials.

That being said, let's jump straight into installing MESSAGEix through Anaconda
```